hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalMapStatsImpl.java

```diff
@@ -539,9 +539,9 @@ public String toString() {
                 + ", getCount=" + getCount
                 + ", putCount=" + putCount
                 + ", removeCount=" + removeCount
-                + ", totalGetLatencies=" + NANOSECONDS.toMillis(totalGetLatenciesNanos)
-                + ", totalPutLatencies=" + NANOSECONDS.toMillis(totalPutLatenciesNanos)
-                + ", totalRemoveLatencies=" + NANOSECONDS.toMillis(totalRemoveLatenciesNanos)
+                + ", totalGetLatencies=" + NANOSECONDS.toMillis(totalGetLatencyNanos)
```

Comment:
Probably it also makes sense to rename the labels in the `toString` representation. It's unlikely that someone is relying on them.